### PR TITLE
Fills in the Team page

### PR DESCRIPTION
**Pull request by swilliams**
Includes:

* Roles
* Advancement
* One on Ones
* Diversity

### CODE REVIEW

File: docs/Team.md [hunk 1]
```diff
@@ -6,3 +6,114 @@ title: "Tallwave Guidelines — Team"
...
+### Lead Software Developer
+
+**Traits**
+* Can lead a team of developers. This is different from management.
+* Can pick up a new technology and be productive fairly quickly
+* Does a lot of non-coding stuff such as:
+  * Requirements gathering
+  * Customer management
+  * Sales support
+* Manages clients, coaches them, and get’s them on board with the Goal for the engagement.
```
Comment (krantzinator) [inline docs/Team.md]: typo - 'gets'

File: docs/Team.md [hunk 2]
```diff
@@ -6,3 +6,114 @@ title: "Tallwave Guidelines — Team"
...
+**Day to Day Examples**
+
+A Lead Dev will have a high degree of ownership in their stack. They’ll be able to help write out plans for how work should be done and set goals for a group to work towards.
+
+Leads mentor more inexperienced developers more often. This could be answering questions, pair programming, or running code reviews. Leads help Software Developers grow to Seniors and Seniors to Leads.
+
+They have more conversations with clients. Per the description, Leads will be presenting project plans to clients, getting their buy-in, and working with the PM team to keep them apprised every step of the way.
+
+## Beyond Lead
+This hasn’t been figured out yet. One answer beyond “Lead” is “Management” however just because someone is a great developer doesn’t mean they’ll be a good manager (or even a mediocre one), so it is important to have some sort of advancement plan that does not involve transitioning to management.
```
Comment (krantzinator) [inline docs/Team.md]: add semi-colon: `“Management”;`

File: docs/Team.md [hunk 3]
```diff
@@ -6,3 +6,114 @@ title: "Tallwave Guidelines — Team"
...
+
+## 1:1s
+You will be having a [one-on-one](http://randsinrepose.com/archives/the-update-the-vent-and-the-disaster/) meeting regularly with your manager. This should be every week, barring illness or holiday. Half an hour is the norm, but these can definitely stretch out to longer should the need arise. 
+
+In the one-on-one you'll discuss how you're feeling. This is not a status update, though it may relate to the status of a project. How are you feeling? Happy? Stressed? Are you freaking out about something in particular?
+
+We'll also have conversations about various topics and ask specific questions, like ["If you were CEO of this company, what would you change?"](https://jasonevanish.com/2014/05/29/101-questions-to-ask-in-1-on-1s/). This is so that you and your manager can get to get to know each other a little bit better and also to keep things fresh, you never know what might come out of that conversation.
```
Comment (krantzinator) [inline docs/Team.md]: `fresh. You`

File: docs/Team.md (hunk 1 above)
Comment (swilliams) [reply docs/Team.md]: I always get that one wrong.

**Comment by swilliams** [comment]
Thanks @krantzinator! Updated.

**Comment by swilliams** [comment]
Created an issue (#22) to track.

File: docs/Team.md [hunk 4]
```diff
@@ -6,3 +6,114 @@ title: "Tallwave Guidelines — Team"
...
+
+We can say that whatever role this is will be much more highly consultative. The knowledge that this individual has is incredibly useful and will be of great importance to our clients. They'll be able to tackle problems like "design a resilient micro-service architecture on AWS that can handle a few million sessions per month and can scale way beyond that." without breaking a sweat.
+
+## Advancement
+You've heard it said that you should dress for the job you want, well similarly, if you want to be a Senior or a Lead developer, you should already be demonstrating those traits. In other words if you ask for a promotion, it shouldn't cause your manager to hesitate.
+
+You should also have a plan and some goals to be working towards, even if you are not looking for a promotion. Goals are tricky. Like estimating software in general, it's quite easy to overestimate how much you can accomplish in a year's time. And in our business, client work has a tendency to take up as much free time as possible, so while special consideration will be given if it's been a particularly busy time, we need to strive to make forward progress.
```
Comment (mmislam101) [inline docs/Team.md]: Suggest a separate section with some tips and methods on how we set goals and deliver.

**Pull request merged**